package.json

```diff
@@ -23,19 +23,23 @@
         "crc": "^3.8.0",
         "escape-html": "^1.0.3",
         "fs-extra": "^8.1.0",
+        "glob-promise": "^3.4.0",
         "merge": "^1.2.1",
         "mkdirp": "^0.5.1",
+        "promisepipe": "^3.0.0",
         "qs": "^6.7.0",
+        "tmp-promise": "^2.0.2",
         "unzipper": "^0.10.3",
```

Comment:
Couldn't find any use of this package so it should be removed.

Reply:
tmp-promise is used in the ContentTypeInformationRepository, the Editor and the PackageManager to create temporary files and directories (in the system temp dir)

Reply:
Sorry about that. Somehow my search didn't find it but now it's there.